Squablr/Views/CardView.m

```diff
@@ -25,6 +25,89 @@ - (instancetype)initWithFrame:(CGRect)frame {
     return self;
 }
 
+- (instancetype)initWithProfile:(CGRect)frame profile:(Profile *)profile {
+    self = [super initWithFrame:frame];
+    if (self) {
+        [self setup];
+        self.profile = profile;
+        // Creating UIImage view programmatically
+        PFImageView *imageView = [[PFImageView alloc] initWithFrame:CGRectMake(0, 0, 350, 500)];
```

Comment:
Don't worry about this for now since we have other things to prioritize but you really want to use auto layout on this image view instead of manually setting the frame like this